.github/pull_request_template.md

```diff
@@ -3,8 +3,14 @@ closes #
 
 ### Checklist
 - [ ] There are no warnings in the browser console
+- [ ] I made sure the component is WCAG compliant using ANDI
+- [ ] I made sure the UI follows the style guide in the wireframe (font, font-size, etc...)
+- [ ] I made sure that the screen reader can read all the text
+- [ ] I made sure I removed all the console.logs
```

Comment:
This can be a lint rule

Reply:
good idea

Reply:
pls review again: added Eslint rule and removed unnecessary code